Dockerfile

```diff
@@ -96,6 +96,12 @@ RUN set -x \
        && cp "$GOPATH"/bin/crictl /usr/bin/ \
        && rm -rf "$GOPATH"
 
+# Install conmon
+RUN VERSION=v2.0.0 &&\
```

Comment:
Do we want to use a more recent version here?

Reply:
I am thinking a separate PR for that, this is already pretty bloated

Reply:
SGTM